yt/frontends/flash/data_structures.py

```diff
@@ -400,7 +400,7 @@ def _parse_parameter_file(self):
         dle = np.array([self.parameters[f"{ax}min"] for ax in "xyz"]).astype("float64")
         dre = np.array([self.parameters[f"{ax}max"] for ax in "xyz"]).astype("float64")
         if self.dimensionality < 3:
-            for d in [dimensionality] + list(range(3 - dimensionality)):
+            for d in range(self.dimensionality, 3):
```

Comment:
We changed to using `self.dimensionality` -- this ok?

I tested with this script:

```python
def old_way(dimensionality):
    print(f"Old way: {dimensionality}")
    for d in [dimensionality] + list(range(3 - dimensionality)):
        print(f"dimension: {d}")

def new_way(dimensionality):
    print(f"New way: {dimensionality}")
    for d in range(dimensionality, 3):
        print(f"dimension: {d}")

old_way(1)
new_way(1)
print()
old_way(2)
new_way(2)
print()
old_way(3)
new_way(3)
```

and got:

```
Old way: 1
dimension: 1
dimension: 0
dimension: 1
New way: 1
dimension: 1
dimension: 2

Old way: 2
dimension: 2
dimension: 0
New way: 2
dimension: 2

Old way: 3
dimension: 3
New way: 3
```

So I think this is right?

Reply:
looks right to me